control/control.SLED.xml

```diff
@@ -822,6 +822,16 @@ Please visit us at http://www.suse.com/.
                     <enable_next>yes</enable_next>
                 </module>
                 <module>
+                    <name>complex_welcome</name>
```

Comment:
Hm, that will add the complete dialog which license, language selection, etc... this does not look correct to me.

I think for AY there should be a dedicated small client which just displays the BETA (when present) and nothing more.

Reply:
Oh it's skipped in AY mode: https://github.com/yast/yast-installation/blob/04df92a2a827ac859870e86833aa856dc91a7107/src/lib/installation/clients/inst_complex_welcome.rb#L47

Reply:
Please add a comment that in AY mode it only displays a Beta warning and does nothing else. It is a bit confusing that the entry is very same as in regular installation but the behavior is completely different.